Classes/Authentication/TokenAuthenticationProvider.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pixelant\PxaSightRegistrationApi\Authentication;
+
+use http\Exception\InvalidArgumentException;
+use Psr\Http\Message\ServerRequestInterface;
+
+class TokenAuthenticationProvider implements AuthenticationProviderInterface
+{
+
+    /**
+     * @param ServerRequestInterface $request
+     * @return bool
+     */
+    public function authenticate(ServerRequestInterface $request): bool
+    {
+       $serverParams = $request->getServerParams();
+       $serverToken = getenv('TOKEN');
```

Comment:
Make the variable name more unique so it doesn't get changed accidentally:

```suggestion
       $serverToken = getenv('APP_SIGHTREG_API_TOKEN');
```